code/modules/clothing/gloves/stungloves.dm

```diff
@@ -6,7 +6,7 @@
 	action_button_name = "Toggle Power Glove"
 	price_tag = 500
 	var/stunforce = 0
-	var/agonyforce = 50
+	var/agonyforce = 20
```

Comment:
This is just plain bad, I will need 5 hits with gloves to down someone. Make it 30 at least.

Reply:
Done